sa/sa.go

```diff
@@ -428,21 +426,7 @@ func (ssa *SQLStorageAuthority) CountCertificatesByNames(ctx context.Context, do
 }
 
 func (ssa *SQLStorageAuthority) CountCertificatesByExactNames(ctx context.Context, domains []string, earliest, latest time.Time) ([]*sapb.CountByNames_MapElement, error) {
```

Comment:
Can you file a ticket to remove this RPC.

Reply:
https://github.com/letsencrypt/boulder/issues/4310